lib/manifestTools/assets/web-manifest-extended.json

```diff
@@ -11,7 +11,43 @@
       "items": {
         "$ref": "#/definitions/mjs_access_whitelist_rule"
       }
-    }
+    },
+    "mjs_extended_scope": {

```

Comment:
I find it a bit hard to understand (from the name) how this related to scope? Extended to what? What is the result of adding things here? @marcoscaceres @boyofgreen